src/App.css

```diff
@@ -35,6 +35,18 @@ h1, h2, h3, h4, h5, h6, span {
 #qcard {
   overflow: hidden;
 }
+#categories{
+  font-family: "Lora", serif;
+  font-size: 24px;
+  color: white;
+  background-color: #88a65e;
+  height: fit-content;
+  text-align: center;
+  padding: 10px;
+}
+#home,#education,#it,#vehicles,#others{
```

Comment:
This last line can be written like `#categories a {` because they are all `a` (links) within #categories.

Reply:
Yes! much better ;-)